LambdaS3FileZipper.App/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading.Tasks;
+using Amazon;
+using Amazon.Runtime;
+using Amazon.S3;
+using LambdaS3FileZipper.Aws;
+using LambdaS3FileZipper.Models;
+using Serilog;
+
+namespace LambdaS3FileZipper.App
+{
+    internal class Program
+    {
+		internal static async Task Main()
+        {
+			// Enable Serilog Console logging
+			Log.Logger = new LoggerConfiguration().MinimumLevel.Verbose().WriteTo.Console().CreateLogger();
+
+			// Todo: Fill in credential, request details
+			var accessKey = "access-key";
+			var secretKey = "secret-key";
+			var endpoint = RegionEndpoint.USEast1;
+			var requestGuid = "guid";
+			var request = new Request
+			{
+				OriginBucketName = "origin-bucket",
+				OriginResourceName = $"{requestGuid}/results",
```

Comment:
Internal implementation details are leaking ;)

Reply:
@brianfeucht, I know 😏 I kinda left that to make it easier to test but not give too much up 😉. I **should** look into another non-`Console` option, tho, so thanks for the tip! 👍